frontline-functions-quickstart/.env.example

```diff
@@ -0,0 +1,35 @@
+# description: Twilio phone number to send SMS from. Your patient will see this number.
+# format: phone_number
+# required: true
+# configurable: true
+TWILIO_PHONE_NUMBER=
+
+# description: (Optional) A Whatsapp enabled phone number we'll use as an example customer phone number to test the app end-to-end
+# format: phone_number
+# required: false
+TWILIO_WHATSAPP_NUMBER=
+
+# description: Agent username: this is the username of one of the users added to the Identity Provider (SSO). We'll use this to log into the app and each example customer we'll create will be assigned to this agent.
+# format: text
+# required: true
+AGENT_USERNAME=
```

Comment:
we haven't actually agreed on what to use but we use `agent`, `worker` and `user` for the same entity. In demo service code mainly `worker` is used for naming variables and function, it might be better to use worker here as well.